contracts/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "@openzeppelin/contracts": "^5.0.1",
+    "@openzeppelin/contracts-upgradeable": "^5.0.1",
```

Comment:
We'll have conflicts with openzeppelin version beween this and https://github.com/omni-network/omni/pull/288